### DISCUSSION TITLE
minecraft stopped working on synology

### DISCUSSION THREAD

**Discussion by McFcologne** [itzg/docker-minecraft-server#3133] [Q&A] [answered]
Im using minecraft for a while on my synology and it all worked fine for a long time. Lately it started crashing after attempted update while starting the container:

2024/11/02 10:12:59 | stdout | Double check that the given VERSION is valid
2024/11/02 10:12:59 | stdout | ERROR failed to download from https://www.minecraft.net/bedrockdedicatedserver/bin-linux/bedrock-server-1.21.44.01.zip
2024/11/02 10:12:59 | stdout | curl: (92) HTTP/2 stream 1 was not closed cleanly: INTERNAL_ERROR (err 2)
2024/11/02 10:12:59 | stdout | Downloading Bedrock server version 1.21.44.01 ...
2024/11/02 10:12:57 | stdout | Looking up latest version...
2024/11/02 10:12:57 | stdout | [37mDEBU[0m[0000] Resolved GID=100 from match path
2024/11/02 10:12:57 | stdout | [37mDEBU[0m[0000] Resolved UID=1026 from match path
2024/11/02 10:12:57 | stdout | [37mDEBU[0m[0000] Using /data to match uid and gid

When im refering to the possible root cause:
curl: (92) HTTP/2 stream 1 was not closed cleanly: INTERNAL_ERROR (err 2)

i found, that i could check forcing http 1.1 with the curl parameter --http1.1 ... but what do i need to modify to do this?

Or is there anything else i could check?

**Comment by itzg** [comment, marked as answer]
Re-pull the image and it will be fixed. 

You actually posted in the wrong repo and this is the issue you're experiencing https://github.com/itzg/docker-minecraft-bedrock-server/issues/458
